### PR TITLE
Variables: Adds user interaction reports

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
This PR adds reporting for some variable related user interactions. One of the goals with this is to figure out how much the variable inspection is used.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by joshhunt — APPROVED** [review]
:]

File: public/app/features/variables/editor/VariableEditorList.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { VariableIdentifier } from '../state/types';
 import { UsagesToNetwork, VariableUsageTree } from '../inspect/utils';
 import { VariableEditorListRow } from './VariableEditorListRow';
 import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import { reportInteraction } from '../../../../../packages/grafana-runtime';
```
Comment (ashharrison90) [inline public/app/features/variables/editor/VariableEditorList.tsx:10]: `@grafana/runtime`

File: public/app/features/variables/editor/VariableEditorListRow.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ import { hasOptions, isAdHoc, isQuery } from '../guard';
 import { toVariableIdentifier, VariableIdentifier } from '../state/types';
 import { VariableUsagesButton } from '../inspect/VariableUsagesButton';
 import { VariableModel } from '../types';
+import { reportInteraction } from '../../../../../packages/grafana-runtime';
```
Comment (ashharrison90) [inline public/app/features/variables/editor/VariableEditorListRow.tsx:13]: `@grafana/runtime`

File: public/app/features/variables/inspect/VariableUsagesButton.tsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { IconButton } from '@grafana/ui';
 
 import { UsagesToNetwork } from './utils';
 import { NetworkGraphModal } from './NetworkGraphModal';
+import { reportInteraction } from '../../../../../packages/grafana-runtime';
```
Comment (ashharrison90) [inline public/app/features/variables/inspect/VariableUsagesButton.tsx:6]: `@grafana/runtime`

File: public/app/features/variables/inspect/VariablesDependenciesButton.tsx [hunk 4]
```diff
@@ -6,6 +6,7 @@ import { createDependencyEdges, createDependencyNodes, filterNodesWithDependenci
 import { store } from '../../../store/store';
 import { VariableModel } from '../types';
 import { NetworkGraphModal } from './NetworkGraphModal';
+import { reportInteraction } from '../../../../../packages/grafana-runtime';
```
Comment (ashharrison90) [inline public/app/features/variables/inspect/VariablesDependenciesButton.tsx:9]: `@grafana/runtime`

**Review by ashharrison90 — CHANGES_REQUESTED** [review]
lgtm, just needs the imports fixing 🙌

**Comment by hugohaggmark** [comment]
Thanks for the feedback @ashharrison90, so sorry for the import mess 🙈

File: public/app/features/variables/editor/VariableEditorList.tsx (hunk 1 above)
Comment (hugohaggmark) [reply public/app/features/variables/editor/VariableEditorList.tsx]: Fixed!

File: public/app/features/variables/editor/VariableEditorListRow.tsx (hunk 2 above)
Comment (hugohaggmark) [reply public/app/features/variables/editor/VariableEditorListRow.tsx]: Fixed!

File: public/app/features/variables/inspect/VariableUsagesButton.tsx (hunk 3 above)
Comment (hugohaggmark) [reply public/app/features/variables/inspect/VariableUsagesButton.tsx]: Fixed!

File: public/app/features/variables/inspect/VariablesDependenciesButton.tsx (hunk 4 above)
Comment (hugohaggmark) [reply public/app/features/variables/inspect/VariablesDependenciesButton.tsx]: Fixed!

**Review by ashharrison90 — APPROVED** [review]
no worries. lgtm! 🙌

**Pull request merged**